### PR TITLE
Improve Dutch translation

### PR DESCRIPTION
**Pull request by Daan4**
I have spent some time improving the Dutch translation. Since I am not a professional translator it is far from perfect, however it should be a lot better than the generated text. If another Dutch speaking person could look it over that would be ideal.

In the Dutch language it is common to use english loan words if it makes sense, so I've opted to use 'Block',  'Node', and 'Exchange' for example as the English word as I felt it makes sense and will be commonly understood. Ledger I've replaced by 'Database', but otherwise 'Ledger' would be more acceptable than the direct translation. Some other 'nano jargon' I found hard to translate in a sensible way as well so it might be worth considering using the English words for more terms than I did.

If further improvement is required later on (or new strings to translate are added) feel free to send me a message.

Also I did not test this file with the application in any way, so be sure to check that I did not mess up the syntax.

### CODE REVIEW

File: src/i18n/locales/nl.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
 {
   "common": {
-    "blockExplorer": "Blokkeer Explorer",
+    "blockExplorer": "block Explorer",
```
Comment (tombertrand) [inline src/i18n/locales/nl.json:3]: This one would need to be capitalized

File: src/i18n/locales/nl.json [hunk 2]
```diff
@@ -33,78 +33,78 @@
...
+    "votingWeight": "Een account met een minimum van {{minWeight}} NANO of >= 0,1% van het online stemgewicht eraan gedelegeerd, is vereist om de status van hoofdvertegenwoordiger te krijgen. Indien geconfigureerd op een stemmende node, worden de stemmen die de node genereert opnieuw uitgezonden door andere nodes die de stemmen ontvangen, waardoor het netwerk sneller consensus kan bereiken.",
+    "pending": "Een transactiestatus waarbij een block dat geld verzendt, is gepubliceerd en bevestigd door het netwerk, maar een overeenkomend block dat het geld ontvangt, nog niet is verzonden of bevestigd.",
+    "nakamotoCoefficient": "De Nakamoto-coëfficiënt is een manier om de decentralisatie van een cryptovaluta of ander gedecentraliseerd systeem te kwantificeren. Het is het aantal entiteiten dat nodig is hebt om ten minste één essentieel subsysteem in gevaar te brengen. Voor Nano zou meer dan 67% van het online stemgewicht van hoofdvertegenwoordigers in gevaar moeten worden gebracht.",
```
Comment (tombertrand) [inline src/i18n/locales/nl.json:66]: interesting, might have to change that one on other languages too, 50% -> 67%, will wait a few days tho to be sure it's the final direction its headed

File: src/i18n/locales/nl.json (hunk 1 above)
Comment (Daan4) [reply src/i18n/locales/nl.json]: Whoops, I think a find and replace messed that one up. I checked some more and fixed a couple other capitalization mistakes as well.

File: src/i18n/locales/nl.json (hunk 2 above)
Comment (Daan4) [reply src/i18n/locales/nl.json:66]: Yes I took the liberty to change it to 67% since that seems to be the current default as of 22.0.

Comment (tombertrand) [reply src/i18n/locales/nl.json:66]: will use a variable in there in an upcoming PR

**Pull request merged**